tests/unit/conditions/test_core_shims.py

```diff
@@ -13,7 +13,7 @@ def test_conditional_message_kit_serialization(enacted_federated_policy, ERC1155
     )
     mk_bytes = bytes(mk)
     restored_mk = MessageKit.from_bytes(mk_bytes)
-    assert restored_mk.lingos is None
+    assert restored_mk.conditionss is None
```

Comment:
```suggestion
    assert restored_mk.conditions is None
```

Reply:
Yep - see https://github.com/nucypher/nucypher/pull/2961#discussion_r957407413 in #2961 . Good eye 👍